src/utils.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2025, Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { SfProject } from '@salesforce/core';
+import { findAuthoringBundle } from '@salesforce/agents';
+
+export const findAuthoringBundleInProject = (project: SfProject, apiName: string): string | undefined => {
```

Comment:
This method is already defined in the `agents` library, and should be updated/added there - I think that method needs an update anyways, it probably uses `aiAuthoringBundle`, no `s`, in its search